SimTKcommon/include/SimTKcommon/internal/Serialize.h

```diff
@@ -176,6 +176,14 @@ template <class T> inline bool
 readUnformatted(std::istream& in, T& v) {
     String token;
     if (!readOneTokenUnformatted(in, token)) return false;
+	for (int i = 0; i < token.size(); i++) {
```

Comment:
Can you explain me what you are doing? Are you replacing brackets by empty spaces? Why? When is this needed and problematic? Thanks for clarifying.

Reply:
I observed issues when getting the inertia vector (vector of 6 elements), surprisingly not with Vectors of 3 elements. In vectors of 6 elements, Recorder values were between parenthesis and not identified as Vector of six elements. I just overcome this issue by removing these parentheses.